index.js

```diff
@@ -771,6 +804,11 @@ instance.prototype.actions = function () {
 		for (s in self.sources) {
 			self.sourcelist.push({ id: s, label: s })
 		}
+		for (s in self.scenes) {
+			let scene = { id: s, label: s }
+			if (self.sourcelist.some(sourcelist => sourcelist.id === scene.id) === false)
+			self.sourcelist.push({ id: s, label: s })
```

Comment:
Does `scene` need to be created? I don't see where it's used. It looks like the condition after the `if` should be indented.

Reply:
Just tested it, you're right it would be simply to just directly use `s`. Also, fixed the indentation! 
https://github.com/bitfocus/companion-module-obs-studio/pull/100/commits/5b27d8becb97498ff5712c0276c362578339ea3e